test/e2e-go/upgrades/rekey_support_test.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"github.com/algorand/go-algorand/data/basics"
+	"github.com/algorand/go-algorand/protocol"
 	"github.com/algorand/go-algorand/test/framework/fixtures"
 )
 
```

Comment:
Typo in the comment below: 
tests that **we** rekey -> tests that **the** rekey

Reply:
I'll make that change before merging the code in if it would pass travis..